### PR TITLE
add docs for phoneNumber signup

### PR DESCRIPTION
**Pull request by prateekmadhikar**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -117,6 +117,61 @@ After the signup is successful, the 'success' callback will be called with infor
...
+  }
+}
+```
+
+This call will take the provided user data, and use it to authenticate the current user and add them to your brand's contact list in Hive.
+
+For ensuring that a user is reachable via their Phone Number, it is recommended to also ask for or enforce a country code whenever possible. In cases where Hive cannot resolve a country from a number, a default of `+1` (United States/Canada) will be assumed for the country code.
+
+For legal purposes, be sure to collect the <code>didSmsOptIn</code> intent from the user as this will decide whether or not you will be able to send SMS to them. If this flag is passed in as <code>false</code>, or if this information is not collected at all, the user will still be imported into your Hive brand but no SMS messages can be targeted at them.
+
```
Comment (prateekmadhikar) [inline source/index.html.md]: i added this, the rest is copypasta more or less

**Comment by prateekmadhikar** [comment]
[add phoneNumberSignup docs to sdk.hive.co](https://trello.com/c/u5Uyo5JZ/9656-add-phonenumbersignup-docs-to-sdkhiveco)

File: source/index.html.md (hunk 1 above)
Comment (patmanh) [reply source/index.html.md]: took a quick pass for explicitness/brevity - thanks for writing this!

```
To ensure that phone numbers are properly saved, it is recommended to provide phone numbers in E.164 or RFC3966 format. In cases where Hive cannot resolve a poorly formatted phone number, a default country code of `+1` (United States/Canada) will be assumed.

To ensure you can legally send SMS messages to your contacts via Hive, you must also collect explicit opt-in consent from each user that complies with TCPA. If users have provided you with explicit consent to receive SMS messages, you should pass a value of <code>true</code> for the <code>didSmsOptIn</code> parameter. If this flag is passed in as <code>false</code> (or if this information is not collected at all) the user will still be imported into your Hive account but no SMS messages can be targeted at them.
```

**Review by patmanh — COMMENTED** [review]
@prateekmadhikar quick copy pass then LGTM!

**Pull request merged**